packages/open-next/src/core/routing/i18n/index.ts

```diff
@@ -52,6 +85,70 @@ export function localizePath(internalEvent: InternalEvent): string {
   if (isLocalizedPath(internalEvent.rawPath)) {
     return internalEvent.rawPath;
   }
+
   const detectedLocale = detectLocale(internalEvent, i18n);
+
   return `/${detectedLocale}${internalEvent.rawPath}`;
 }
+
+export function handleLocaleRedirect(
```

Comment:
Add a one line to explain when it returns false vs InternalResult (cloud be on `@return`)